smartseq2_single_sample/pipelines/hisat2_QC_pipeline.wdl

```diff
@@ -0,0 +1,87 @@
+import "hisat2.wdl" as hisat2
+import "picard.wdl" as picard
+## QC pipeline
```

Comment:
I think you should be a bit more verbose about why you do this as opposed to explaining what is done, because it seems a bit crazy to run alignment twice without understanding that we need to do it for consistency with picard's expectations about BAM formats. 

It also has the potential to help engineers who might have ideas about how to fix this problem that we haven't thought about.

Reply:
added some comments in other wdl file and hope it will help